src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RSAPKCS1KeyExchangeFormatter.cs

```diff
@@ -26,6 +26,11 @@ public override string Parameters
             }
         }
 
+        public RandomNumberGenerator Rng {
+            get { return null; }
+            set { throw new PlatformNotSupportedException(); }
```

Comment:
My thought is since these values are never used, the implementation of CreateKeyExchange shouldn't need to change. @bartonjs any thoughts on this?